Source/WebCore/inspector/InspectorInstrumentation.cpp

```diff
@@ -930,8 +958,14 @@ void InspectorInstrumentation::addMessageToConsoleImpl(InstrumentingAgents& inst
     MessageType type = message->type();
     String messageText = message->message();
 
-    if (auto* consoleAgent = instrumentingAgents.webConsoleAgent())
+    if (auto* consoleAgent = instrumentingAgents.webConsoleAgent()) {
+#if ENABLE(WEBDRIVER_BIDI)
+        for (const InspectorInstrumentationConsoleMessageObserver& observer : consoleMessageObservers())
+            observer(*message);
```

Comment:
for example, why is this dependent on whether there's a `WebConsoleAgent`?  by this point we already have the `ConsoleMessage` so it shouldn't matter if it exists since this is for WebDriver, not Web Inspector